scripts/flow/environment.js

```diff
@@ -70,50 +70,73 @@ declare module 'EventListener' {
 declare function __webpack_chunk_load__(id: string): Promise<mixed>;
 declare function __webpack_require__(id: string): any;
 
-declare module 'fs/promises' {
-  declare var access: (path: string, mode?: number) => Promise<void>;
-  declare var lstat: (
-    path: string,
-    options?: ?{bigint?: boolean},
-  ) => Promise<mixed>;
-  declare var readdir: (
-    path: string,
+declare module 'fs' {
+  declare function existsSync(path: string): boolean;
```

Comment:
Kind of unfortunate that we're not relying on built-in definitions here. Maybe we can just get rid of this when we update Flow and get Promises from upstream.

Reply:
So there are built in definitions that could be used but there is no promises definition for fs.promises.lstat, fs.promises.stat does not define an option, and the default fs.promises.realpath is ambiguous.

If there is an easy way to overload specific entries with flow? I couldn't figure out how to do that, but if we could overload just those three functions we could avoid the rest of the noise